src/hepattn/models/maskformer.py

```diff
@@ -234,12 +273,12 @@ def loss(self, outputs: dict, targets: dict) -> dict:
                     outputs[layer_name][task.name][output_name] = outputs[layer_name][task.name][output_name][batch_idxs, pred_idxs]
 
         # Compute the losses for each task in each block
-        losses = {}
+        losses: dict[str, dict[str, Any]] = {}
```

Comment:
I think we can replace `Any` with `Tensor`? probably better to be specific if we are adding type hints